x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -72,6 +72,63 @@ setup:
   - match: { "test-index.mappings.properties.sparse_field.model_settings.task_type": sparse_embedding }
   - length: { "test-index.mappings.properties.sparse_field": 3 }
 
+---
+"Indexes sparse vector document - inference metadata fields":
+  - requires:
+      test_runner_features: [ capabilities ]
+      reason: Test targets semantic text with inference metadata fields
+      capabilities:
+        - method: GET
+          path: /_inference
+          capabilities: [ inference_metadata_fields ]
+
+  # Checks mapping is not updated until first doc arrives
+  - do:
+      indices.get_mapping:
+        index: test-index
+
+  - match: { "test-index.mappings.properties.sparse_field.type": semantic_text }
+  - match: { "test-index.mappings.properties.sparse_field.inference_id": sparse-inference-id }
+  - length: { "test-index.mappings.properties.sparse_field": 2 }
+
+  - do:
+      index:
```

Comment:
Could you please help me understand why the index command breaks everything out explicitly rather than just indexing `these are not the droids you're looking for. He's free to go around` in `sparse_field`?

Reply:
This test suite is explicitly testing the format used by `_inference_fields` to represent inference results. As such, breaking out the format is by design here.